apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/RdaServerJob.java

```diff
@@ -186,6 +192,16 @@ private Config(
       }
     }
 
+    private RdaService.Version createVersion() {
+      RdaService.Version.VersionBuilder versionBuilder = RdaService.Version.builder();
+      if (serverMode == ServerMode.S3) {
+        versionBuilder.version(String.format("S3:%d", System.currentTimeMillis()));
+      } else {
+        versionBuilder.version(String.format("Random:%d", randomSeed));
+      }
```

Comment:
Would we ever want to search on all S3 for example? Would it be beneficial to break this into multiple columns?

Reply:
This was discussed during refinement and it was decided to keep this implementation simple until we know the extra complexity is needed.  Specifically this implementation would allow us to search for all S3 claims.

Reply:
We can still do this with a LIKE statement, it may not be fast, but it would work as a one off in TEST/SBX